samples/language/Tuple.qs

```diff
@@ -14,10 +14,40 @@ function Main() : (Int, String) {
     let myTuple = ("Id", 0, 1.);
     Message($"Tuple: {myTuple}");
 
+    // A tuple may be unpacked by assigning it to a tuple of variables
+    let (name, n, d) = myTuple;
+    Message($"Unpacked: {name}, {n}, {d}");
+
+    // If not all items are needed, a tuple may be unpacked into
+    // a tuple of variables with a wildcard `_` for the unwanted items.
+    let (name, _, _) = myTuple;
+    Message($"Name: {name}");
+
+    // A "tuple" may be created with a single item, but it is treated
+    // as the item itself enclosed in parenthesis, which is called
+    // singleton tuple equivalence.
```

Comment:
I think we should avoid mentions of singleton tuple equivalence, since we don't technically have that anymore. Classic QDK would treat `3`, `(3)`, and `(3,)` as the same value of type `Int` because it had true singleton tuple equivalence (there was no such thing as a tuple of arity 1 as it always got turned into a singleton). With the modern QDK we actually do support tuples of arity 1 if they explicitly made a tuple with the trailing comma like the example below this shows.

Likewise, the comment at the top of the file that says "Tuples containing a single item are treated as identical to the item itself, both in tpye and value, which is called singleton tuple equivalence" should be removed.